packages/icon-button-react/src/Icons/IconCalendar.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
Dette ikonet finnes allerede i datepicker-react.

Kanskje tid for en ikon pakke så vi slipper å duplisere oss selv?

Reply:
Jeg planlegger å bruke IconButton på datepicker og andre relevante komponenter. Sånn, skal duplikasjoner bli fjernet.